packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -434,20 +426,20 @@ const toRelate: Array<relation> = [
  * Attaches each attribute struct to it's corresponding class using the ID look up
  * table from 'createStructuresById'
  *
- * @param classStructs
+ * @param classStructsById
  * @param attributeStructs
- * @param lookupTable
  */
 
-function attachClassAttributes(classStructs, attributeStructs, lookupTable) {
+function attachClassAttributes(classStructsById, attributeStructs) {
   //initialize all of the class structs to have no attributes
-  classStructs.forEach(struct => {
+  Object.values(classStructsById).forEach(struct => {
+    // $FlowFixMe
```

Comment:
Flow fought with me pretty hard here. Think it's related to https://github.com/facebook/flow/issues/2221.